docs/user/general/datatypes.rst

```diff
@@ -104,7 +104,11 @@ The table below list the mapping between OpenSearch Data Type, OpenSearch SQL Da
 | nested          | array               | STRUCT    |
 +-----------------+---------------------+-----------+
 
-Notes: Not all the OpenSearch SQL Type has correspond OpenSearch Type. e.g. data and time. To use function which required such data type, user should explicitly convert the data type.
+Notes:
+* Not all the OpenSearch SQL Type has correspond OpenSearch Type. e.g. data and time. To use function which required such data type, user should explicitly convert the data type.
+* date*: Maps to `timestamp` by default. Based on the "format" property `date` can map to `date` or `time`. See list of supported named formats [here](https://opensearch.org/docs/latest/field-types/supported-field-types/date/).
```

Comment:
It is not an MD file, rst has another link syntax.

Reply:
Changed in https://github.com/Bit-Quill/opensearch-project-sql/pull/268/commits/2672a79de79e905fe8a5fb7f8dd18ab4879f30af